applications/nrf_desktop/bootloader_dfu.rst

```diff
@@ -185,6 +189,20 @@ For an example of a bootloader Kconfig configuration file defined by the applica
   Both mentioned firmware upgrade methods are not used simultaneously by any of the configurations.
   For example, the ``nrf52840dk/nrf52840`` board in ``mcuboot_smp`` file suffix uses only the background DFU and does not enable the serial recovery feature.
 
+MCUboot bootloader on nRF54L
+----------------------------
+
+The nRF54L SoC Series enhances security and reduces boot times by using hardware cryptography in the MCUboot immutable bootloader.
+The |NCS| allows using hardware cryptography for ED25519 signature (``SB_CONFIG_BOOT_SIGNATURE_TYPE_ED25519``) on the nRF54L SoC Series.
+
+You can enhance security further by enabling the following sysbuild Kconfig options:
```

Comment:
Aren't those options enabled by default?

Reply:
No. Application needs to enable them in sysbuild configuration